testdata/baselines/reference/submoduleAccepted/conformance/checkJsdocSatisfiesTag10.errors.txt.diff

```diff
@@ -2,19 +2,16 @@
 +++ new.checkJsdocSatisfiesTag10.errors.txt
 @@= skipped -0, +0 lines =@@
 -/a.js(6,5): error TS2353: Object literal may only specify known properties, and 'x' does not exist in type 'Partial<Record<Keys, unknown>>'.
++/a.js(6,5): error TS2353: Object literal may only specify known properties, and 'x' does not exist in type 'Partial<Record<"a" | "b" | "c" | "d", unknown>>'.
  /a.js(14,11): error TS2339: Property 'd' does not exist on type '{ a: number; b: string; x: number; }'.
 
 
--==== /a.js (2 errors) ====
-+==== /a.js (1 errors) ====
-     /** @typedef {"a" | "b" | "c" | "d"} Keys */
-     
-     const p = /** @satisfies {Partial<Record<Keys, unknown>>} */ ({
-         a: 0,
+@@= skipped -9, +9 lines =@@
          b: "hello",
          x: 8 // Should error, 'x' isn't in 'Keys'
--        ~
+         ~
 -!!! error TS2353: Object literal may only specify known properties, and 'x' does not exist in type 'Partial<Record<Keys, unknown>>'.
++!!! error TS2353: Object literal may only specify known properties, and 'x' does not exist in type 'Partial<Record<"a" | "b" | "c" | "d", unknown>>'.
```

Comment:
Weird, where did `a b c d` come from?

Reply:
Expansion of `Keys` - presumably a difference in type node printback behavior to do with not currently having most input node reuse logic in place right now.